### PR TITLE
Rebase PR#10798 Update standard EventContent definitions

### PR DESCRIPTION
**Pull request by jeongeun**
"Fix Conflicts"

### CODE REVIEW

**Comment by jeongeun** [comment]
merge

**Comment by cvuosalo** [comment]
@cmsbuild please test

File: RecoLocalCalo/Configuration/python/RecoLocalCalo_EventContentCosmics_cff.py [hunk 1]
```diff
@@ -27,9 +27,13 @@
...
                                            'keep HORecHitsSorted_horecoMB_*_*',
                                            'keep HFRecHitsSorted_hfrecoMB_*_*',
-                                           'keep ZDCDataFramesSorted_*Digis_*_*',
+                                           #'keep ZDCDataFramesSorted_*Digis_*_*',
+                                           'keep ZDCDataFramesSorted_hcalDigis_*_*',
+                                           'keep ZDCDataFramesSorted_simHcalUnsuppressedDigis_*_*',
                                            'keep ZDCRecHitsSorted_*_*_*',
-                                           'keep HcalUnpackerReport_*_*_*'
+                                           #'keep HcalUnpackerReport_*_*_*'
+                                           'keep HcalUnpackerReport_castorDigis_*_*'
```
Comment (slava77) [inline RecoLocalCalo/Configuration/python/RecoLocalCalo_EventContentCosmics_cff.py]: there is a comma missing here.
@jeongeun please double-check all other files and update only after ```runTheMatrix.py -l limited -i all``` completes

**Comment by cvuosalo** [comment]
@jeongeun:
Please fix the problems with this pull request.

**Comment by jeongeun** [comment]
Fix the missing comma and
'runTheMatrix.py -l limited -i all' is completed.
Please test

**Comment by slava77** [comment]
@cmsbuild please test

**Comment by slava77** [comment]
``` git rebase -i ``` can be used to reduce the number of commits by squashing the intermediate ones

**Comment by slava77** [comment]
the squashing would be nice, but is not absolutely necessary; just trying to keep the history shorter

**Comment by cvuosalo** [comment]
@jeongeun:
Your changes have altered the event content, which is not correct. The following collections are missing:

floats_generalTracks_MVAValues
recoPFTauDiscriminator_hpsPFTauDiscriminationByRawPileupWeightedIsolation3Hits
recoPFTauDiscriminator_hpsPFTauDiscriminationByPhotonPtSumOutsideSignalCone
recoPFTauDiscriminator_hpsPFTauDiscriminationByLoosePileupWeightedIsolation3Hits
recoPFTauDiscriminator_hpsPFTauNeutralIsoPtSum
recoPFTauDiscriminator_hpsPFTauDiscriminationByTightPileupWeightedIsolation3Hits
recoPFTauDiscriminator_hpsPFTauDiscriminationByMediumPileupWeightedIsolation3Hits
recoPFTauDiscriminator_hpsPFTauPUcorrPtSum
recoPFTauDiscriminator_hpsPFTauChargedIsoPtSum

And recoPhotons_gedPhotonsTmp has been incorrectly added.

Please fix these problems.

**Comment by jeongeun** [comment]
@cvuosalo 
I included the missing parts of hpsPFTau* and deleted the gedPhotonTmp lines.
However in generalTracks_MVAValues case,

When I checked the event content definitions at 7_6_0_pre2
I found the float type generalTracks Module has different Label name "MVAVals" as follow
Type                                                    Module                        Label             Process
---------------------------------------------------------------------------------------------------------------
edm::ValueMap<float>                  "generalTracks"             "MVAVals"         "RECO"

so I changed the Label name "MVAValues" to "MVAVals" in my PR.
Is that label newly updated or changed again? or Do I misunderstand something here?
Please let me know, Thank you.

**Comment by cvuosalo** [comment]
@cmsbuild please test

**Comment by cvuosalo** [comment]
@jeongeun:
In the latest version of 76X, CMSSW_7_6_X_2015-09-12-1100, generalTracks_MVAValues is present. This pull request should not remove generalTracks_MVAValues from the event content.

**Comment by jeongeun** [comment]
Yes. I fixed it (not remove generalTracks_MVAValues)
and I'll keep check the event contents in the latest version of 76X. Thank you.
please test

**Comment by cvuosalo** [comment]
@cmsbuild please test

**Comment by cvuosalo** [comment]
+1

For #11068 17a054f45f53fb34e7c03e714625b7dce7c23c8c

Expanding wildcards to be more specific for event content statements. There should be no change in run-time behavior or resulting event content.

The code changes are satisfactory, and Jenkins [tests]() against baseline CMSSW_7_6_X_2015-09-13-1100 show no significant differences, as expected. An extended test of workflow 25202.0_TTbar_13 with 70 events against baseline CMSSW_7_6_0_pre4 shows no significant differences and no change in event content.

**Comment by slava77** [comment]
@monttj @diguida @mmusich 
please check this PR, your signature is needed
Thank you.

**Comment by slava77** [comment]
@monttj @diguida @mmusich
please check this PR, your signature is needed
Thank you.

**Comment by monttj** [comment]
+1
It is to make the output contents more specific in AT side.

**Comment by diguida** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**